src/referencePackages/src/microsoft.codeanalysis.workspaces.common/4.0.1/Customizations.props

```diff
@@ -0,0 +1,16 @@
+<Project>
+
+  <ItemGroup>
+    <!-- Humanizer is provided through source-build-externals -->
+    <ProjectReference Remove="../../humanizer.core/2.2.0/Humanizer.Core.2.2.0.csproj" />
+
+    <!-- Upgrade System.Composition to a version that has netstandard2.0 -->
```

Comment:
This comment should clarify why 9.0.0 versus the minimum version that support netstandard2.0.  I think it would be helpful to call out that the metadata is eliminated.